src/routes/v1/class.route.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const validate = require('../../middlewares/validate');
+const classValidation = require('../../validations/class.validation');
+const classController = require('../../controllers/class.controller');
+const auth = require('../../middlewares/auth');
+
+const router = express.Router();
+
+router.post('/createClass', validate(classValidation.createClass), classController.createClass)
```

Comment:
in the REST Standard the create route is alwayys the plural of the model name
Create: `post('/classes')`
getAll: `get('/classes')`
getOne: `get('/classes/:id')`
edit: `patch('/classes/:id')`
delete: delete('/classes/:id')

Reply:
so this would just become `router.post('/', ....`

Reply:
ok, i will change it